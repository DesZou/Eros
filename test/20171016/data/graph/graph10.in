50 1225 55
1 2 470
1 3 359
1 4 342
1 5 825
1 6 777
1 7 712
1 8 13
1 9 901
1 10 23
1 11 111
1 12 48
1 13 654
1 14 737
1 15 484
1 16 707
1 17 504
1 18 769
1 19 944
1 20 620
1 21 570
1 22 6
1 23 461
1 24 943
1 25 108
1 26 553
1 27 2
1 28 279
1 29 348
1 30 119
1 31 502
1 32 610
1 33 426
1 34 216
1 35 314
1 36 604
1 37 245
1 38 936
1 39 348
1 40 430
1 41 285
1 42 188
1 43 1
1 44 152
1 45 826
1 46 719
1 47 563
1 48 909
1 49 884
1 50 109
2 3 955
2 4 799
2 5 349
2 6 752
2 7 113
2 8 523
2 9 381
2 10 545
2 11 10
2 12 86
2 13 548
2 14 305
2 15 961
2 16 695
2 17 405
2 18 872
2 19 113
2 20 983
2 21 121
2 22 679
2 23 913
2 24 950
2 25 233
2 26 345
2 27 849
2 28 432
2 29 256
2 30 199
2 31 796
2 32 704
2 33 336
2 34 780
2 35 343
2 36 836
2 37 388
2 38 309
2 39 569
2 40 985
2 41 742
2 42 742
2 43 507
2 44 952
2 45 219
2 46 905
2 47 922
2 48 113
2 49 403
2 50 656
3 4 227
3 5 575
3 6 517
3 7 701
3 8 46
3 9 77
3 10 890
3 11 952
3 12 782
3 13 231
3 14 945
3 15 904
3 16 424
3 17 939
3 18 883
3 19 366
3 20 546
3 21 209
3 22 47
3 23 703
3 24 94
3 25 874
3 26 985
3 27 816
3 28 366
3 29 917
3 30 431
3 31 822
3 32 562
3 33 64
3 34 781
3 35 286
3 36 761
3 37 940
3 38 249
3 39 51
3 40 28
3 41 625
3 42 537
3 43 431
3 44 185
3 45 517
3 46 571
3 47 684
3 48 790
3 49 935
3 50 381
4 5 490
4 6 492
4 7 25
4 8 294
4 9 4
4 10 751
4 11 509
4 12 805
4 13 710
4 14 251
4 15 718
4 16 473
4 17 618
4 18 161
4 19 290
4 20 425
4 21 219
4 22 234
4 23 567
4 24 45
4 25 897
4 26 83
4 27 956
4 28 980
4 29 711
4 30 233
4 31 490
4 32 799
4 33 41
4 34 647
4 35 859
4 36 822
4 37 442
4 38 366
4 39 385
4 40 922
4 41 709
4 42 85
4 43 522
4 44 194
4 45 570
4 46 849
4 47 714
4 48 850
4 49 435
4 50 710
5 6 617
5 7 593
5 8 647
5 9 31
5 10 335
5 11 921
5 12 759
5 13 770
5 14 742
5 15 933
5 16 456
5 17 895
5 18 743
5 19 742
5 20 64
5 21 155
5 22 423
5 23 958
5 24 548
5 25 680
5 26 251
5 27 97
5 28 665
5 29 511
5 30 393
5 31 512
5 32 810
5 33 105
5 34 497
5 35 816
5 36 74
5 37 538
5 38 645
5 39 371
5 40 34
5 41 255
5 42 357
5 43 270
5 44 277
5 45 825
5 46 965
5 47 866
5 48 65
5 49 91
5 50 331
6 7 958
6 8 193
6 9 881
6 10 419
6 11 677
6 12 873
6 13 665
6 14 814
6 15 972
6 16 306
6 17 318
6 18 379
6 19 415
6 20 351
6 21 522
6 22 820
6 23 5
6 24 919
6 25 916
6 26 820
6 27 913
6 28 275
6 29 35
6 30 135
6 31 234
6 32 514
6 33 162
6 34 155
6 35 389
6 36 212
6 37 836
6 38 934
6 39 993
6 40 756
6 41 265
6 42 416
6 43 931
6 44 985
6 45 346
6 46 426
6 47 452
6 48 718
6 49 505
6 50 743
7 8 696
7 9 313
7 10 325
7 11 594
7 12 104
7 13 473
7 14 989
7 15 368
7 16 738
7 17 76
7 18 290
7 19 871
7 20 174
7 21 47
7 22 26
7 23 384
7 24 941
7 25 999
7 26 402
7 27 1
7 28 374
7 29 736
7 30 924
7 31 525
7 32 41
7 33 652
7 34 306
7 35 136
7 36 583
7 37 342
7 38 641
7 39 134
7 40 673
7 41 856
7 42 297
7 43 963
7 44 176
7 45 129
7 46 308
7 47 56
7 48 429
7 49 339
7 50 965
8 9 693
8 10 899
8 11 412
8 12 380
8 13 802
8 14 793
8 15 339
8 16 319
8 17 67
8 18 4
8 19 398
8 20 346
8 21 707
8 22 621
8 23 682
8 24 82
8 25 151
8 26 167
8 27 738
8 28 238
8 29 974
8 30 803
8 31 339
8 32 479
8 33 696
8 34 575
8 35 632
8 36 66
8 37 733
8 38 404
8 39 473
8 40 260
8 41 887
8 42 431
8 43 729
8 44 54
8 45 107
8 46 283
8 47 344
8 48 136
8 49 900
8 50 845
9 10 198
9 11 194
9 12 928
9 13 888
9 14 850
9 15 537
9 16 607
9 17 218
9 18 354
9 19 983
9 20 465
9 21 607
9 22 163
9 23 471
9 24 24
9 25 796
9 26 76
9 27 399
9 28 790
9 29 446
9 30 390
9 31 166
9 32 207
9 33 12
9 34 470
9 35 520
9 36 306
9 37 114
9 38 630
9 39 200
9 40 196
9 41 768
9 42 944
9 43 5
9 44 6
9 45 659
9 46 669
9 47 792
9 48 337
9 49 202
9 50 222
10 11 901
10 12 119
10 13 435
10 14 161
10 15 592
10 16 339
10 17 87
10 18 213
10 19 10
10 20 163
10 21 635
10 22 201
10 23 700
10 24 797
10 25 329
10 26 762
10 27 891
10 28 184
10 29 822
10 30 84
10 31 877
10 32 812
10 33 861
10 34 260
10 35 723
10 36 765
10 37 580
10 38 923
10 39 991
10 40 332
10 41 30
10 42 725
10 43 191
10 44 62
10 45 473
10 46 202
10 47 723
10 48 961
10 49 376
10 50 381
11 12 548
11 13 869
11 14 651
11 15 828
11 16 365
11 17 263
11 18 285
11 19 783
11 20 455
11 21 604
11 22 239
11 23 260
11 24 23
11 25 299
11 26 428
11 27 961
11 28 698
11 29 875
11 30 365
11 31 364
11 32 301
11 33 48
11 34 597
11 35 440
11 36 230
11 37 896
11 38 271
11 39 897
11 40 999
11 41 675
11 42 343
11 43 802
11 44 836
11 45 698
11 46 542
11 47 389
11 48 994
11 49 143
11 50 2
12 13 667
12 14 907
12 15 974
12 16 565
12 17 52
12 18 55
12 19 370
12 20 221
12 21 681
12 22 925
12 23 786
12 24 324
12 25 712
12 26 914
12 27 101
12 28 249
12 29 223
12 30 497
12 31 69
12 32 679
12 33 821
12 34 349
12 35 424
12 36 116
12 37 286
12 38 803
12 39 524
12 40 884
12 41 509
12 42 675
12 43 132
12 44 47
12 45 831
12 46 47
12 47 782
12 48 369
12 49 21
12 50 645
13 14 200
13 15 744
13 16 938
13 17 34
13 18 26
13 19 371
13 20 494
13 21 48
13 22 500
13 23 545
13 24 122
13 25 881
13 26 339
13 27 912
13 28 53
13 29 956
13 30 545
13 31 735
13 32 295
13 33 784
13 34 763
13 35 390
13 36 780
13 37 215
13 38 87
13 39 740
13 40 115
13 41 85
13 42 525
13 43 745
13 44 231
13 45 256
13 46 589
13 47 645
13 48 373
13 49 834
13 50 129
14 15 481
14 16 671
14 17 940
14 18 485
14 19 493
14 20 1
14 21 446
14 22 244
14 23 249
14 24 178
14 25 261
14 26 604
14 27 528
14 28 414
14 29 524
14 30 606
14 31 66
14 32 65
14 33 518
14 34 863
14 35 777
14 36 710
14 37 170
14 38 314
14 39 390
14 40 785
14 41 608
14 42 337
14 43 325
14 44 978
14 45 4
14 46 17
14 47 462
14 48 475
14 49 89
14 50 222
15 16 85
15 17 843
15 18 127
15 19 631
15 20 804
15 21 748
15 22 31
15 23 332
15 24 736
15 25 42
15 26 406
15 27 629
15 28 57
15 29 966
15 30 977
15 31 404
15 32 678
15 33 161
15 34 561
15 35 501
15 36 368
15 37 989
15 38 447
15 39 101
15 40 166
15 41 864
15 42 865
15 43 313
15 44 258
15 45 642
15 46 549
15 47 41
15 48 761
15 49 522
15 50 37
16 17 429
16 18 56
16 19 355
16 20 523
16 21 150
16 22 813
16 23 5
16 24 223
16 25 112
16 26 268
16 27 817
16 28 776
16 29 190
16 30 571
16 31 669
16 32 920
16 33 519
16 34 808
16 35 881
16 36 539
16 37 536
16 38 233
16 39 265
16 40 680
16 41 794
16 42 573
16 43 108
16 44 864
16 45 429
16 46 177
16 47 796
16 48 391
16 49 573
16 50 781
17 18 255
17 19 249
17 20 108
17 21 432
17 22 709
17 23 639
17 24 720
17 25 273
17 26 459
17 27 586
17 28 22
17 29 104
17 30 480
17 31 319
17 32 647
17 33 132
17 34 774
17 35 676
17 36 690
17 37 271
17 38 299
17 39 818
17 40 707
17 41 247
17 42 312
17 43 371
17 44 106
17 45 437
17 46 474
17 47 582
17 48 615
17 49 891
17 50 279
18 19 280
18 20 21
18 21 482
18 22 451
18 23 261
18 24 124
18 25 946
18 26 700
18 27 408
18 28 699
18 29 605
18 30 34
18 31 703
18 32 637
18 33 438
18 34 716
18 35 211
18 36 540
18 37 310
18 38 671
18 39 822
18 40 684
18 41 952
18 42 736
18 43 430
18 44 185
18 45 862
18 46 555
18 47 409
18 48 115
18 49 58
18 50 461
19 20 713
19 21 793
19 22 297
19 23 111
19 24 570
19 25 76
19 26 453
19 27 852
19 28 193
19 29 950
19 30 155
19 31 357
19 32 960
19 33 626
19 34 349
19 35 954
19 36 679
19 37 989
19 38 528
19 39 205
19 40 109
19 41 829
19 42 192
19 43 290
19 44 424
19 45 705
19 46 986
19 47 857
19 48 545
19 49 946
19 50 626
20 21 799
20 22 363
20 23 329
20 24 967
20 25 225
20 26 685
20 27 873
20 28 636
20 29 498
20 30 990
20 31 893
20 32 512
20 33 592
20 34 849
20 35 989
20 36 552
20 37 546
20 38 249
20 39 637
20 40 83
20 41 380
20 42 503
20 43 631
20 44 535
20 45 159
20 46 993
20 47 403
20 48 562
20 49 82
20 50 837
21 22 59
21 23 467
21 24 333
21 25 375
21 26 692
21 27 722
21 28 680
21 29 135
21 30 122
21 31 94
21 32 656
21 33 335
21 34 346
21 35 886
21 36 466
21 37 3
21 38 718
21 39 409
21 40 746
21 41 273
21 42 13
21 43 353
21 44 262
21 45 3
21 46 817
21 47 917
21 48 30
21 49 803
21 50 709
22 23 559
22 24 55
22 25 774
22 26 415
22 27 54
22 28 841
22 29 916
22 30 338
22 31 316
22 32 718
22 33 88
22 34 207
22 35 794
22 36 380
22 37 19
22 38 821
22 39 235
22 40 900
22 41 997
22 42 46
22 43 271
22 44 315
22 45 430
22 46 572
22 47 697
22 48 265
22 49 714
22 50 196
23 24 106
23 25 25
23 26 76
23 27 681
23 28 623
23 29 184
23 30 716
23 31 820
23 32 457
23 33 298
23 34 875
23 35 955
23 36 412
23 37 233
23 38 655
23 39 128
23 40 306
23 41 231
23 42 394
23 43 756
23 44 433
23 45 94
23 46 48
23 47 52
23 48 626
23 49 749
23 50 657
24 25 162
24 26 619
24 27 462
24 28 915
24 29 422
24 30 167
24 31 838
24 32 292
24 33 607
24 34 388
24 35 69
24 36 966
24 37 860
24 38 813
24 39 719
24 40 236
24 41 624
24 42 426
24 43 938
24 44 516
24 45 126
24 46 493
24 47 433
24 48 70
24 49 985
24 50 102
25 26 207
25 27 243
25 28 128
25 29 794
25 30 280
25 31 145
25 32 631
25 33 142
25 34 277
25 35 295
25 36 857
25 37 298
25 38 598
25 39 420
25 40 933
25 41 932
25 42 590
25 43 933
25 44 199
25 45 859
25 46 89
25 47 702
25 48 656
25 49 222
25 50 433
26 27 64
26 28 288
26 29 587
26 30 553
26 31 908
26 32 571
26 33 84
26 34 681
26 35 79
26 36 442
26 37 927
26 38 180
26 39 647
26 40 452
26 41 740
26 42 486
26 43 413
26 44 13
26 45 533
26 46 705
26 47 987
26 48 37
26 49 89
26 50 50
27 28 200
27 29 811
27 30 161
27 31 128
27 32 2
27 33 452
27 34 112
27 35 973
27 36 563
27 37 711
27 38 420
27 39 825
27 40 103
27 41 739
27 42 362
27 43 610
27 44 228
27 45 475
27 46 777
27 47 384
27 48 171
27 49 793
27 50 332
28 29 752
28 30 238
28 31 408
28 32 430
28 33 527
28 34 547
28 35 222
28 36 834
28 37 106
28 38 760
28 39 914
28 40 121
28 41 828
28 42 423
28 43 947
28 44 392
28 45 864
28 46 684
28 47 874
28 48 352
28 49 751
28 50 564
29 30 104
29 31 207
29 32 979
29 33 233
29 34 250
29 35 624
29 36 637
29 37 212
29 38 516
29 39 54
29 40 861
29 41 146
29 42 644
29 43 223
29 44 704
29 45 137
29 46 257
29 47 663
29 48 766
29 49 708
29 50 108
30 31 738
30 32 148
30 33 510
30 34 165
30 35 38
30 36 600
30 37 508
30 38 961
30 39 879
30 40 572
30 41 230
30 42 514
30 43 573
30 44 124
30 45 279
30 46 640
30 47 420
30 48 651
30 49 362
30 50 647
31 32 259
31 33 179
31 34 384
31 35 560
31 36 903
31 37 843
31 38 895
31 39 765
31 40 752
31 41 926
31 42 85
31 43 231
31 44 843
31 45 786
31 46 62
31 47 130
31 48 778
31 49 958
31 50 986
32 33 540
32 34 562
32 35 800
32 36 595
32 37 978
32 38 627
32 39 197
32 40 493
32 41 523
32 42 645
32 43 106
32 44 934
32 45 137
32 46 513
32 47 963
32 48 144
32 49 149
32 50 597
33 34 290
33 35 141
33 36 370
33 37 493
33 38 602
33 39 571
33 40 741
33 41 334
33 42 708
33 43 440
33 44 259
33 45 484
33 46 674
33 47 479
33 48 956
33 49 327
33 50 146
34 35 411
34 36 431
34 37 217
34 38 310
34 39 340
34 40 861
34 41 611
34 42 721
34 43 264
34 44 866
34 45 65
34 46 124
34 47 956
34 48 642
34 49 731
34 50 659
35 36 291
35 37 88
35 38 726
35 39 393
35 40 949
35 41 869
35 42 474
35 43 83
35 44 723
35 45 895
35 46 84
35 47 531
35 48 523
35 49 96
35 50 631
36 37 596
36 38 688
36 39 505
36 40 711
36 41 622
36 42 459
36 43 158
36 44 999
36 45 186
36 46 247
36 47 179
36 48 915
36 49 856
36 50 825
37 38 583
37 39 864
37 40 302
37 41 247
37 42 400
37 43 43
37 44 325
37 45 484
37 46 624
37 47 591
37 48 119
37 49 321
37 50 79
38 39 883
38 40 203
38 41 989
38 42 928
38 43 67
38 44 332
38 45 991
38 46 224
38 47 621
38 48 182
38 49 667
38 50 461
39 40 319
39 41 923
39 42 268
39 43 811
39 44 512
39 45 309
39 46 562
39 47 829
39 48 852
39 49 773
39 50 172
40 41 409
40 42 716
40 43 700
40 44 7
40 45 93
40 46 845
40 47 223
40 48 127
40 49 777
40 50 703
41 42 480
41 43 229
41 44 965
41 45 289
41 46 579
41 47 378
41 48 309
41 49 979
41 50 926
42 43 652
42 44 666
42 45 917
42 46 93
42 47 735
42 48 194
42 49 750
42 50 685
43 44 221
43 45 546
43 46 151
43 47 431
43 48 552
43 49 220
43 50 465
44 45 551
44 46 295
44 47 444
44 48 996
44 49 845
44 50 831
45 46 887
45 47 282
45 48 43
45 49 164
45 50 494
46 47 555
46 48 297
46 49 929
46 50 711
47 48 266
47 49 606
47 50 741
48 49 677
48 50 431
49 50 128
50
14 21
25 38
26 49
31 35
45 42
12 39
45 21
4 13
45 15
38 6
26 19
33 14
2 20
22 30
20 11
7 46
14 47
49 8
14 9
48 32
21 17
42 16
50 13
43 30
36 11
9 21
1 35
45 11
38 9
10 25
35 12
34 14
13 38
32 11
7 46
36 8
33 47
39 18
5 37
1 20
47 25
45 42
17 33
42 26
43 45
41 22
17 5
15 23
34 21
32 8
