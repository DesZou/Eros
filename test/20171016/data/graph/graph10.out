115
197
148
210
130
194
63
134
128
199
150
105
143
133
139
107
146
134
89
114
153
162
169
138
205
84
154
142
168
194
184
181
161
130
107
144
158
171
112
90
228
130
143
128
82
111
162
173
182
69
